src/wp-admin/security.php

```diff
@@ -0,0 +1,188 @@
+		<p><?php _e( 'As ClassicPress continues to evolve, the Security page will become the hub for all security features for ClassicPress core and 3<sup>rd</sup> party plugins that choose to support it.' ); ?></p>
+		<p>
+		<?php
+		printf(
+			/* translators: canonical link to security forum */
+			__( 'Watch this page and the ClassicPress <a href="%s" rel="noopener" target="_blank">Security forum</a> for more news as development continues.' ),
+			'https://link.classicpress.net/forum/security'
+		);
+		?>
+		</p>
+	</div>
+
+		<?php
+		break;
+	case 'developers':
+		/**
+		 * hilite.me
+		 *   + The opening PHP tag is required to get the highlighting to work, but it's been removed here for clarity
+		 *   + The "default" colour scheme was used
+		 */
+		?>
+
+	<div id="poststuff">
+		<div id="post-body" class="metabox-holder columns-2">
+			<div id="post-body-content">
+				<div class="meta-box-sortables ui-sortable">
+					<div class="postbox">
+						<h2><span><?php _e( 'Overview' ); ?></span></h2>
+						<div class="inside">
+							<div>
+								<p><?php _e( 'The Security page is the first visible step in improving the overall approach to security in ClassicPress. Its pupose is to solve two related problems: <strong>discovery</strong> and <strong>auditing</strong>.' ); ?></p>
+								<p><?php _e( 'Most plugins are organised around their core purpose; after all, that purpose is the reason they were installed. Unfortunately this leads to poor discoverability of security-related settings - they may be in there somewhere, there may be none at all - without looking through everything there&lsquo;s no way to know.' ); ?></p>
+								<p><?php _e( 'Currently, auditing the overall security profile of a ClassicPress site is impractical if there are more than a few plugins. Having all security-related settings from all plugins in one place means those settings can be audited far more easily, as the time taken will be proportional to the number of settings, not the number of places to look for those settings.' ); ?></p>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class="meta-box-sortables ui-sortable">
+					<div class="postbox">
+						<h2><span><?php _e( 'API' ); ?></span></h2>
+						<div class="inside">
+							<p><?php _e( 'There is just one new function: <code>add_security_page()</code>. It works similarly to the other <code>add_xxx_page()</code> helpers, with two important changes:' ); ?></p>
+							<p>
+								<ul class="ul-disc">
+									<li><?php _e( 'there is no <code>$capability</code> argument - it is always <code>manage_options</code>' ); ?></li>
+									<li><?php _e( 'the <code>$menu_slug</code> must match an active plugin slug.' ); ?></li>
+								</ul>
+							</p>
+							<p>The function also adds a link to your security page to the plugin actions. [This needs to be phrased better - brain fog, suggestions please.]</p>
```

Comment:
How about: The function also adds a link from your plugin's row in the plugins list directly to your security settings page.